src/structs/0728.hpp

```diff
@@ -4312,574 +4312,574 @@ struct sP_LS2CL_REP_SHARD_LIST_INFO_SUCC {
-static_assert(sizeof(sItemVendor) == 28 || sizeof(sItemVendor) == 0);
-static_assert(sizeof(sEmailInfo) == 204 || sizeof(sEmailInfo) == 0);
-static_assert(sizeof(sQuickSlot) == 4 || sizeof(sQuickSlot) == 0);
-static_assert(sizeof(sItemInven) == 16 || sizeof(sItemInven) == 0);
-static_assert(sizeof(sRunningQuest) == 52 || sizeof(sRunningQuest) == 0);
-static_assert(sizeof(sSYSTEMTIME) == 32 || sizeof(sSYSTEMTIME) == 0);
-static_assert(sizeof(sTimeLimitItemDeleteInfo2CL) == 8 || sizeof(sTimeLimitItemDeleteInfo2CL) == 0);
-static_assert(sizeof(sEPRecord) == 6 || sizeof(sEPRecord) == 0);
-static_assert(sizeof(sNanoBank) == 4 || sizeof(sNanoBank) == 0);
-static_assert(sizeof(sTimeBuff_Svr) == 32 || sizeof(sTimeBuff_Svr) == 0);
-static_assert(sizeof(sBulletAppearanceData) == 20 || sizeof(sBulletAppearanceData) == 0);
-static_assert(sizeof(sTransportationLoadData) == 32 || sizeof(sTransportationLoadData) == 0);
-static_assert(sizeof(sAttackResult) == 24 || sizeof(sAttackResult) == 0);
-static_assert(sizeof(sCAttackResult) == 40 || sizeof(sCAttackResult) == 0);
-static_assert(sizeof(sSkillResult_Damage) == 20 || sizeof(sSkillResult_Damage) == 0);
-static_assert(sizeof(sSkillResult_DotDamage) == 32 || sizeof(sSkillResult_DotDamage) == 0);
-static_assert(sizeof(sSkillResult_Heal_HP) == 16 || sizeof(sSkillResult_Heal_HP) == 0);
-static_assert(sizeof(sSkillResult_Heal_Stamina) == 16 || sizeof(sSkillResult_Heal_Stamina) == 0);
-static_assert(sizeof(sSkillResult_Stamina_Self) == 24 || sizeof(sSkillResult_Stamina_Self) == 0);
-static_assert(sizeof(sSkillResult_Damage_N_Debuff) == 32 || sizeof(sSkillResult_Damage_N_Debuff) == 0);
-static_assert(sizeof(sSkillResult_Buff) == 16 || sizeof(sSkillResult_Buff) == 0);
-static_assert(sizeof(sSkillResult_BatteryDrain) == 40 || sizeof(sSkillResult_BatteryDrain) == 0);
-static_assert(sizeof(sSkillResult_Damage_N_Move) == 36 || sizeof(sSkillResult_Damage_N_Move) == 0);
-static_assert(sizeof(sSkillResult_Move) == 24 || sizeof(sSkillResult_Move) == 0);
-static_assert(sizeof(sSkillResult_Resurrect) == 12 || sizeof(sSkillResult_Resurrect) == 0);
-static_assert(sizeof(sPC_HP) == 8 || sizeof(sPC_HP) == 0);
-static_assert(sizeof(sPC_BATTERYs) == 12 || sizeof(sPC_BATTERYs) == 0);
-static_assert(sizeof(sPC_NanoSlots) == 16 || sizeof(sPC_NanoSlots) == 0);
-static_assert(sizeof(sPC_Nano) == 12 || sizeof(sPC_Nano) == 0);
-static_assert(sizeof(sNPCLocationData) == 24 || sizeof(sNPCLocationData) == 0);
-static_assert(sizeof(sGroupNPCLocationData) == 44 || sizeof(sGroupNPCLocationData) == 0);
-static_assert(sizeof(sPCGroupMemberInfo) == 112 || sizeof(sPCGroupMemberInfo) == 0);
-static_assert(sizeof(sNPCGroupMemberInfo) == 32 || sizeof(sNPCGroupMemberInfo) == 0);
-static_assert(sizeof(sEPElement) == 36 || sizeof(sEPElement) == 0);
-static_assert(sizeof(sChannelInfo) == 8 || sizeof(sChannelInfo) == 0);
+static_assert(sizeof(sP_CL2FE_REQ_PC_ENTER) == 80 || sizeof(sP_CL2FE_REQ_PC_ENTER) == 0, "");
```

Comment:
?

Any modern C++17-supporting compiler should do fine with what was there?

Reply:
I haven't done that